sycl/unittests/device_code_split/CMakeLists.txt

```diff
@@ -0,0 +1,66 @@
+
+function(add_device_code_split_unittest)
+list(GET ARGV 0 currentTest)
+list(GET ARGV 1 splitFlag)
+list(GET ARGV 2 hostSources)
```

Comment:
[cmake_parse_arguments](https://cmake.org/cmake/help/latest/command/cmake_parse_arguments.html)?

Reply:
thanks, cmake_parse_arguments will be better